app/templates/components/speaker-list.hbs

```diff
@@ -0,0 +1,11 @@
+<h3>{{t 'Speakers'}}</h3>
+<div class="ui grid">
+  {{#each speakers as |speaker|}}
+    <div class="four wide column speaker-item">
+      <img class="ui small picture" src="/images/placeholders/avatar.png">
+      <br>
+      <div>{{speaker.name}}</div>
+      <div>{{speaker.organisation}}</div>
```

Comment:
What about the social links of the PR ? See attached screenshot in the related issue.